setup.py

```diff
@@ -183,6 +183,8 @@
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: Implementation :: CPython',
+        'Programming Language :: Python :: Implementation :: PyPy',
```

Comment:
PyPy is failing in CI, so I think we shouldn't declare PyPy-readiness... Better to delete it for now.

```
    In file included from psycopg/psycopgmodule.c:38:0:

    ./psycopg/error.h:32:5: error: unknown type name ‘PyBaseExceptionObject’

         PyBaseExceptionObject exc;

         ^

    error: command 'cc' failed with exit status 1

    

    ----------------------------------------

Command "/home/travis/virtualenv/pypy-2.6.1/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-X37tln/psycopg2/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/pip-x_RSca-record/install-record.txt --single-version-externally-managed --compile --install-headers /home/travis/virtualenv/pypy-2.6.1/include/site/python2.7/psycopg2" failed with error code 1 in /tmp/pip-build-X37tln/psycopg2/

The command "pip install --no-binary :all: psycopg2" failed. Retrying, 3 of 3.
```